src/__tests__/unit/calculate.test.js

```diff
@@ -0,0 +1,41 @@
+import calculate from '../../logic/calculate';
+
+describe('Validate calculate functionality', () => {
+  const num = { total: null, next: null, operation: null };
+  test('multiplication', () => {
+    let result = calculate(num, '10');
+    result = calculate(result, 'x');
+    result = calculate(result, '5');
+    result = calculate(result, '=');
+    expect(result.total).toBe('50');
```

Comment:
- [ ] As requested by the previous code reviewer, please kindly add tests to simulate user interactions using `React Testing Library`. For example, when you click on a particular set of button, what do you expect the page to render. Checkout this [article](https://www.codecademy.com/courses/learn-react-testing/lessons/react-testing-library/exercises/mimicking-user-interactions)

Reply:
Hello @Mihndim2020 😃 ,

Thank you for sharing the linked article. I managed to implement the test.

Fingers crossed 🍀 🤞🏽  for the next review to carry an approval 😃 .

> HappyCoding